voidcrew/code/game/objects/structures/tables_racks.dm

```diff
@@ -25,20 +18,20 @@
 	else
 		return "<span class='notice'>The top cover is firmly stuck on, but could be <i>pried</i> off with considerable effort.</span>"
 
-/obj/structure/table/wood/reinforced/attackby(obj/item/W, mob/user, params)
-	if(W.tool_behaviour == TOOL_CROWBAR && user.a_intent != INTENT_HELP)
-		if(!W.tool_start_check(user, amount=0))
-			return
+/obj/structure/table/wood/reinforced/crowbar_act(mob/living/user, obj/item/item)
```

Comment:
tool_act()'s are supposed to return TRUE on success so it doesnt continue to smak the table afterwards